Source/WTF/wtf/PlatformHave.h

```diff
@@ -971,6 +971,7 @@
 #define HAVE_CFNETWORK_NSURLSESSION_HSTS_WITH_UNTRUSTED_ROOT 1
 #define HAVE_NSURLSESSION_TASK_DELEGATE 1
 #define HAVE_AUDIO_OBJECT_PROPERTY_ELEMENT_MAIN 1
+#define HAVE_IOKIT_MAIN_PORT 1
```

Comment:
We build WebKit on the current OS (Ventura, 130000) and two down-level OSes (Monterey, 120000 and Big Sur, 110000). Big Sur does not have IOKitMain, and so your change will cause a build failure on that platform. Please use the definition I suggested to you in an earlier review comment.

Reply:
#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000) \
    || PLATFORM(IOS_FAMILY) is literally the condition above all this though.

And the EWS is on a Big Sur build, which compiles as seen by the results.

Reply:
Please please please look at the code surrounding my definition.

Please. 

It says right there it's only if it's >= 12000